pkg/kontext/bundle.go

```diff
@@ -47,19 +47,49 @@ func bundle(directory string) (v1.Layer, error) {
 	buf := bytes.NewBuffer(nil)
 	tw := tar.NewWriter(buf)
 	defer tw.Close()
-
+	var excludedDirs []string = make([]string, 0)
 	err := filepath.Walk(directory,
 		func(path string, fi os.FileInfo, err error) error {
 			if err != nil {
 				return err
 			}
 
 			// Skip anything in the .git directory
-			// TODO(mattmoor): expand this to .gitignore / .dockerignore?
+
 			if fi.IsDir() && filepath.Base(path) == ".git" {
```

Comment:
If we're adding ignore support, I think my preference would be to have it encapsulate and replace this.  We should perhaps have a default ignore list, which includes `.git/`.

Reply:
fixed in the latest push